backends/p4tools/testgen/targets/ebpf/test/EBPFXfail.cmake

```diff
@@ -0,0 +1,45 @@
+# XFAILS: tests that currently fail. Most of these are temporary.
+# ================================================
+
+# The code generated by the compiler is invalid.
+p4tools_add_xfail_reason(
+  "testgen-p4c-ebpf"
+  "Error 2: Failed to build the filter:"
+  lpm_ebpf.p4
```

Comment:
These are compilation failures triggered by P4Testgen. It seems like there are issue with the synthesized control plane interface.

Reply:
Many